spatialos-sdk/src/worker/connection.rs

```diff
@@ -328,30 +325,79 @@ impl Connection for WorkerConnection {
         }
     }
 
-    fn send_command_request(
+    fn send_command_request<C: Component>(
         &mut self,
-        _entity_id: EntityId,
-        _request: CommandRequest,
-        _timeout_millis: Option<u32>,
-        _command_parameters: CommandParameters,
+        entity_id: EntityId,
+        request: C::CommandRequest,
+        timeout_millis: Option<u32>,
+        allow_short_circuit: bool,
     ) -> RequestId<OutgoingCommandRequest> {
-        unimplemented!()
+        let timeout = match timeout_millis {
+            Some(c) => &c,
```

Comment:
I'm surprised this compiles. Neat.

Reply:
I think this is a change with NLL? Since the borrow of `c` is valid for the function scope its perfectly happy 😄

Reply:
It's not NLL related, rather the `&u32` is getting coerced to a `*const u32` (which is one of the few implicit conversions that Rust allows), at which point the lifetimes get thrown away 😉